recipe/meta.yaml

```
@@ -102,8 +101,8 @@ outputs:
       run:
         - {{ pin_subpackage('dlib', exact=True) }}
     test:
```

Looks like `conda-build` treats this as containing `None` instead of an empty `dict` when the portion below is selected out. Snippet below taken [from CI]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=854039&view=logs&j=ac463cd8-2ed1-5bc0-c9e7-7aad5c967a56&t=a6af81fd-07f6-5a97-714a-959591713c6e ):

```Python
  File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 2756, in build
    test_script = output_d.get("test", {}).get("script")
AttributeError: 'NoneType' object has no attribute 'get'
#
```


Maybe we can add a selector here to hide the whole `test` section to resolve the issue

```suggestion
    test:  # [not (aarch64 and cuda_compiler_version != "None")]
```